tests/eligibility/test_geiq.py

```diff
@@ -420,3 +422,44 @@ def test_authored_by_prescriber_or_geiq():
     )
 
     assert [prescriber_diagnosis, geiq_diagnosis_with_hiring] == list(valid_diagnoses)
+
+
+def test_ac_level_annex_coherence():
```

Comment:
Le nom complet facilite pour retrouver le texte en fuzzy-find. :shrug: 
```suggestion
def test_administrativecriteria_level_annex_coherence():
```

Reply:
:+1: J'ai renommé la contrainte également.